website/source/docs/internals/telemetry.html.md

```diff
@@ -63,729 +63,1462 @@ The following sections describe available Vault metrics. The metrics interval ca
-
-This should be monitored and alerted on for overall cluster leadership status
-
-### vault.core.post_unseal
-
-**[G]** Gauge (Milliseconds): Duration of time taken by post-unseal operations handled by Vault core
-
-### vault.core.pre_seal
-
-**[G]** Gauge (Milliseconds): Duration of time taken by pre-seal operations
-
-### vault.core.seal-with-request
-
-**[G]** Gauge (Milliseconds): Duration of time taken by requested seal operations
-
-### vault.core.seal
-
-**[G]** Gauge (Milliseconds): Duration of time taken by seal operations
-
-### vault.core.seal-internal
-
-**[G]** Gauge (Milliseconds): Duration of time taken by internal seal operations
-
-### vault.core.step_down
-
-**[S]** Summary (Milliseconds):Duration of time taken by cluster leadership step downs
-
-This should be monitored and alerted on for overall cluster leadership status
-
-### vault.core.unseal
-
-**[S]** Summary (Milliseconds): Duration of time taken by unseal operations
-
-### vault.runtime.alloc_bytes
-
-**[G]** Gauge (Number of bytes): Number of bytes allocated by the Vault process.
-
-This could burst from time to time, but should return to a steady state value.
-
-### vault.runtime.free_count
-
-**[G]** Gauge (Number of objects): Number of freed objects
-
-### vault.runtime.heap_objects
-
-**[G]** Gauge (Number of objects): Number of objects on the heap
-
-This is a good general memory pressure indicator worth establishing a baseline and thresholds for alerting.
-
-### vault.runtime.malloc_count
-
-**[G]** Gauge (Number of objects): Cumulative count of allocated heap objects
-
-### vault.runtime.num_goroutines
-
-**[G]** Gauge (Number of goroutines): Number of goroutines
-
-This serves as a general system load indicator worth establishing a baseline and thresholds for alerting.
-
-### vault.runtime.sys_bytes
-
-**[G]** Gauge (Number of bytes): Number of bytes allocated to Vault
-
-This includes what is being used by Vault's heap and what has been reclaimed but not given back to the operating system.
-
-### vault.runtime.total_gc_pause_ns
-
-**[S]** Summary (Milliseconds): The total garbage collector pause time since Vault was last started
-
-### vault.runtime.total_gc_runs
-
-**[G]** Gauge (Number of operations): Total number of garbage collection runs since Vault was last started
+<table class="table table-bordered table-striped">
+  <tr>
+    <th>Metric</th>
+    <th>Description</th>
+    <th>Unit</th>
+    <th>Type</th>
+  </tr>
+  <tr>
+    <td>`vault.audit.file.log_request`</td>
+    <td>Duration of time taken by audit log requests for the file based audit device mounted as `file`</td>
+    <td>ms</td>
+    <td>summary</td>
+  </tr>
+
+  <tr>
+    <td>`vault.audit.file.log_response`</td>
+    <td>Duration of time taken by audit log responses for the file based audit device mounted as `file`</td>
+    <td>ms</td>
+    <td>summary</td>
+  </tr>
+</table>
+
+<table class="table table-bordered table-striped">
```

Comment:
Did you mean to start a new table?  If so, some kind of sub-title before starting a new table would be nice. 
Perhaps, you can organize them as:

## Audit Metrics

_table here_
vault.audit.log_request
vault.audit.log_response
vault.audit.log_request_failure
vault.audit.log_request_failure

-> **NOTE:** For each audit device, the metrics appear as `vault.audit.<type>.log_request`.  For example, if a file audit device is enabled, its metrics would be `vault.audit.file.log_request`  and `vault.audit.file.log_response` .

### Vault Core
_table here_
(barriers and core metric)

### Vault Runtime Metrics
 _table here_

## Policy and Token Metrics

...

Reply:
I switched us over to regular table syntax, and added the other fixes suggested above.